frontend/www/js/omegaup/components/problem/Mine.vue

```diff
@@ -20,17 +20,17 @@
       <div class="card-body">
         <div class="row align-items-center mb-3">
           <div class="col-md-8 col-lg-6">
-            <omegaup-autocomplete
-              v-model="keyword"
-              class="form-control"
-              :init="(el) => typeahead.problemTypeahead(el)"
+            <input
+              v-model="currentQuery"
+              class="typeahead form-control"
               :placeholder="T.wordsKeywordSearch"
-            ></omegaup-autocomplete>
+            />
           </div>
           <a
             class="btn btn-primary"
             role="button"
-            :href="keyword ? `/problem/mine/?query=${keyword}` : ''"
+            :class="{ disabled: currentQuery === '' }"
+            :href="currentQuery ? `/problem/mine/?query=${currentQuery}` : ''"
```

Comment:
```suggestion
            :href="currentQuery ? `/problem/mine/?query=${encodeURIComponent(currentQuery})` : ''"
```

welp, esto se me pasó.